codalab/apps/web/templates/web/my/participants.html

```diff
@@ -265,6 +268,28 @@ <h4 class="modal-title">Email Participants</h4>
 <script type="text/javascript">
 {% block js %}
 
+    $('#enable_chagrade').on('click', function() {
+        console.log("Something")
+        $.ajax({
+            type: 'post',
+            url: '/api/competition/{{ competition_id }}/enable_chagrade/',
+            contentType: "application/json",
+            dataType: 'json'
+        })
+            .done(function (data) {
+                $('#enable_chagrade').addClass('disabled')
+                console.log(data)
+                alert('Chagrade features enabled for competition.')
+            })
+            .fail(function (response) {
+                if (response) {
+                    var data = JSON.parse(response.responseText);
+                    console.log(data)
```

Comment:
Are errors displayed nicely?

Reply:
Yes they should be. Examples of messages returned based on case:
- 'Competition is not found or is not accessible!'
- 'Could not find competition: ___'
- 'Chagrade bot already exists'
- 'Not authorized!'